packages/design-preview/src/components/style.scss

```diff
@@ -0,0 +1,159 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+@import '@automattic/typography/styles/fonts';
+
+.design-preview {
+	display: block;
+
+	@include break-small {
+		display: flex;
+		flex-direction: row;
+		gap: 32px;
+	}
+}
+
+.design-preview__sidebar {
+	align-items: center;
+	background: #ffffff;
+	border-bottom: 1px solid rgba( 0, 0, 0, 0.05 );
+	box-shadow: -4px 0 8px rgba( 0, 0, 0, 0.07 );
+	box-sizing: border-box;
+	display: flex;
+	height: 96px;
+	inset-inline-start: 0;
+	inset-inline-end: 0;
+	overflow: auto;
+	padding: 8px 24px 0;
+	position: fixed;
+	top: 0;
+	z-index: z-index( '.is-section-stepper', '.design-preview__sticky-variations' );
+
+	> div {
+		display: none;
+
+		&:first-of-type {
+			margin: 0;
+		}
+
+		h2 {
+			color: var( --studio-gray-100 );
+			display: none;
+			font-size: $font-body-small;
+			font-weight: 500;
+			margin-bottom: 0.5rem;
+		}
+	}
+
+	.design-preview__sidebar-title {
+		h1 {
+			color: var( --studio-gray-100 );
+			font-family: $brand-serif;
+			font-weight: 400;
+			letter-spacing: -0.4px;
+			line-height: 2rem;
+		}
+	}
+
+	.design-preview__sidebar-description {
+		p {
+			color: var( --studio-gray-80 );
+			font-size: 0.875rem;
+			line-height: 1.25rem;
+		}
+	}
+
+	.design-preview__sidebar-variations {
+		display: block;
+
+		.design-preview__sidebar-variations-grid {
+			align-items: center;
+			display: flex;
+
+			.design-preview__style-variation {
+				flex-shrink: 0;
+				width: 100px;
+			}
+		}
+	}
+
+	.design-preview__sidebar-support {
+		p {
+			color: var( --studio-gray-80 );
+			font-size: $font-body-small;
+		}
+	}
+
+	.design-preview__sidebar-features {
+		.design-preview__sidebar-features-list {
+			align-items: flex-start;
+			border-radius: 4px;
+			display: flex;
+			flex-wrap: wrap;
+			gap: 12px 4px;
+
+			> div {
+				background-color: var( --studio-gray-5 );
+				border-radius: 4px;
+				color: var( --studio-gray-80 );
+				font-size: 0.75rem;
+				font-weight: 500;
+				line-height: 1.25rem;
+				padding: 0 10px;
+				text-align: center;
+			}
+		}
+	}
+
+	@include break-small {
+		border: 0;
+		box-shadow: none;
+		display: block;
+		height: auto;
+		overflow: visible;
+		padding: 0;
+		position: relative;
+		width: 280px;
+
+		> div {
+			display: block;
+			margin: 2rem 0 0;
+	
+			h2 {
+				display: block;
+			}
+		}
+
+		.design-preview__sidebar-variations {
+			.design-preview__sidebar-variations-grid {
+				display: grid;
+				gap: 8px;
+				grid-template-columns: repeat( 2, 1fr );
+				margin: 0 -5px;
+
+				.design-preview__style-variation {
+					width: 100%;
```

Comment:
Looks like we need to set the width to `auto` or `unset`. Otherwise, the width of the variations element is larger than the sidebar

![image](https://user-images.githubusercontent.com/13596067/188097860-020b3de5-24c1-40eb-90be-a271bfdd253a.png)